mtags/src/main/scala-2/scala/meta/internal/pc/completions/ArgCompletions.scala

```diff
@@ -19,7 +19,12 @@ trait ArgCompletions { this: MetalsGlobal =>
     val editRange: l.Range =
       pos.withStart(ident.pos.start).withEnd(pos.start).toLsp
     val funPos = apply.fun.pos
-    val method: Tree = typedTreeAt(funPos)
+    val method: Tree = typedTreeAt(funPos) match {
+      case Apply(Block(defParams, app: Apply), _)
```

Comment:
Nit: from my perspective a small comment explaining the case would be really welcome (like the very nice remarks in the `isDefaultArg` function). :slightly_smiling_face: